tensorrt_llm/_torch/attention_backend/sparse/kernel.py

```
@@ -1797,137 +1797,174 @@ def triton_rocket_reduce_scores(
-        stride_factor:
-    int = None,  # for strided memory layout (with layer interleaving), defaults to BLOCK_SIZE
-        layer_id: int = 0,  # for layer interleaving layout
+    req_id: torch.Tensor,  # int32 [num_tokens]
+    block_table: torch.
+    Tensor,  # int32 [num_requests, kv_factor, max_num_blocks_per_req]
+    token_indices: torch.
+    Tensor,  # int32 [num_kv_heads, num_tokens, NUM_TOPK_TOKENS]
+    BLOCK_SIZE: int,
+    NUM_TOPK_TOKENS: int = 2048,
+    BLOCK_N: int = 128,  # tile width along columns
+    stride_factor: int = None,  # elements per block in pool
```

Ruff (0.15.6)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)